test/operations/tst_either.jl

```diff
@@ -117,68 +117,68 @@ end
             @test Augmentor.supports_view(op) === false
             @test Augmentor.supports_stepview(op) === false
             @test Augmentor.supports_permute(op) === false
-            @test @inferred(Augmentor.applyeager(op, img)) == rotl90(rect)
-            @test typeof(Augmentor.applyeager(op, img)) <: OffsetArray
+            @test @inferred(Augmentor.applyeager(op, img, Augmentor.randparam(op, img))) == rotl90(rect) # TODO: update 1.2: type instable if remove Augmentor.randparam
```

Comment:
Type instability is detected here for Julia 1.2; the lucky part is that only the out-most wrapper is affected.

Reply:
Is it `randparam` then?

Reply:
Sounds reasonable, I'll check that; not very familiar debugging this kind of issue.

Reply:
It turns out that the following part doesn't work anymore

https://github.com/Evizero/Augmentor.jl/blob/6055db7493352257ddc7b97d63ecaf8ccb43faea/src/operations/either.jl#L175-L181

a typical error is like
```txt
ERROR: return type OffsetArray{Gray{Normed{UInt8,8}},2,Array{Gray{Normed{UInt8,8}},2}} does not match inferred return type OffsetArray
```